content/docs/latest/spire-helm-charts-hardened-about/upgrading.md

```diff
@@ -0,0 +1,48 @@
+
+---
+title: Upgrading
+short: Upgrading
+description: How to upgrade the SPIRE stack using the Helm charts
+kind: spire-helm-charts-hardened-about
+weight: 100
+aliases:
+    - /docs/latest/helm-charts-hardened/upgrading
+---
+
+# Chart version number meaning
+
+The chart has its own version number following the semver scheme of:
+
+`Major`.`Minor`.`Patch` versioning.
+
+# Prerequisites
+
+## Version upgrades
+We only support upgrading one Major/Minor version at a time. Version skipping isn't supported except for patch versions.
+
+Examples:
+ * Its supported to upgrade from 0.11.1 directly to 0.11.7
+ * Its supported to upgrade from 0.11.2 to 0.12.1
+ * Its not supported to upgrade from 0.11.0 to 0.13.0
+
+## Additional Upgrade Steps
+
+From time to time we may need to have you do some upgrade steps before upgrading. We only will do this
```

Comment:
```suggestion
From time to time, we may need to have you perform some upgrade steps before upgrading. We will only do this
```

Reply:
Sorry, there was some merge conflict bits from when the other pr was still in flight. I just removed all the files added in the other pr since they were out of date here.